CONTRIBUTING.md

```diff
@@ -79,7 +79,13 @@ To build the Presto docs, see the [docs README](presto-docs/README.md).
 
 ### Building the Web UI
 
-The Presto Web UI is composed of several React components and is written in JSX and ES6. This source code is compiled and packaged into browser-compatible JavaScript, which is then checked in to the Presto source code (in the `dist` folder). You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to execute these commands. To update this folder after making changes, simply run:
+The Presto Web UI is composed of several React components and is written in JSX and ES6. This source code is compiled and packaged into browser-compatible JavaScript, which is then checked in to the Presto source code (in the `dist` folder). You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to execute these commands.
+
+Install Node.js from the link above and Yarn can be installed using brew:
```

Comment:
```suggestion
Install Node.js from [Download Node.js](https://nodejs.org/en/download/)

To install Yarn using Homebrew:
```
Minor rephrasing: repeating the download link won't hurt, and suggest changing the Yarn instruction to imperative for consistency with your Node.js instruction. Also separating the two points to start the Yarn instruction on the left margin, to make it less likely for a user to overlook it.